test/gurka/test_gtfs.cc

```diff
@@ -121,19 +121,19 @@ TEST(GtfsExample, WriteGtfs) {
   };
   feed.add_stop(nemo);
   struct gtfs::Stop nemo_egress {
-    .stop_id = stopOneID + "_" + to_string(NodeType::kTransitEgress),
-    .stop_name = gtfs::Text("POINT NEMO"), .coordinates_present = true,
-    .stop_lat = station_one_ll->second.second, .stop_lon = station_one_ll->second.first,
-    .parent_station = stopOneID, .location_type = gtfs::StopLocationType::EntranceExit,
-    .stop_timezone = "America/Toronto", .wheelchair_boarding = "1",
+    .stop_id = stopOneID + "_rotating_door_eh", .stop_name = gtfs::Text("POINT NEMO"),
```

Comment:
decided to use a custom id name rather than what valhalla would generate to increase hardness to input from real gtfs